### DISCUSSION TITLE
将v2rayA当作网关，将路由器dhcp的网关设置成v2rayA的 IP，刚开始正常，使用1-2 小时左右局域网内的设备访问国内网络变慢了

### DISCUSSION THREAD

**Discussion by pigracing** [v2rayA/v2rayA#1530] [Q&A]
将v2rayA当作网关，将路由器dhcp的网关设置成v2rayA的 IP，刚开始正常，使用1-2 小时左右局域网内的设备访问国内网络变慢了，请问在设置上有哪些要注意的吗？谢谢
其中，v2rayA是pve上的一台虚拟机，独立 IP，只运行v2rayA，内存4G。具体配置如下图。
<img width="642" alt="image" src="https://github.com/user-attachments/assets/88af1fd0-be64-442a-a4a3-c5cd8ff4a434">

**Comment by aceslup** [comment]
4G都卡，看来我这边的256M，死得透透也算正常。
